modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMSalesPage.testcase

```diff
@@ -0,0 +1,72 @@
+@component-name = "OSB Site Initializer Partner Portal"
+definition {
+
+	property custom.properties = "feature.flag.LPS-135430=true";
+	property osgi.modules.includes = "osb-site-initializer-partner-portal";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "OSB Site Initializer Partner Portal";
+
+	setUp {
+		task ("Set up instance and sign in") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Add a new user") {
+			JSONUser.addUser(
+				userEmailAddress = "partner@liferay.com",
+				userFirstName = "partner",
+				userLastName = "partner",
+				userScreenName = "partner");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "partner@liferay.com");
+		}
+
+		task ("Create a new Partner Portal Site") {
+			PRMUtils.addPRMSite();
+		}
+
+		task ("Go to Partner Portal and login with the created user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "partner@liferay.com",
+				userLoginFullName = "partner");
+
+			PRMUtils.gotoPRM();
+		}
+
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Partner Portal");
+		}
+	}
+
+	@description = "Verify that a Register a New Deal button is displayed on the banner and the Deal Registration Form page will be opened when click on this button"
+	@priority = "5"
+	test CanAccessDealFormViaBannerButton {
+		task ("Given I go to Sales tab") {
+			PRMNavigator.goToNavItem(navItem = "Sales");
+		}
+
+		task ("And I click on Register a New Deal button on banner") {
+			PRMPages.clickBannerLink(linkText = "Register a New Deal");
+		}
```

Comment:
Hi @heiditrindade, this task could be a 'When', since you are doing an action that you result on the action that you expect on the 'Then' task.